game.go

```diff
@@ -37,11 +38,12 @@ func (c *Client) GameList(queries ...Query) (*GameListResult, error) {
 
 // Returns a channel of games (and errors) which can be iterated
 // over transparently, without manually dealing with pagination.
-func (c *Client) GameListIterate(queries ...Query) *GameListIterator {
+func (c *Client) GameListIterate(reverse bool, queries ...Query) *GameListIterator {
```

Comment:
Direction could also have a method `Next` which handles all of the logic on L71-L87.  I'm thinking something like:

```go
type Direction int

func (d Direction) Next(pager Pager) int {
        switch d {
        case Forward:
                if pager.Limit > pager.From {
                        return 0
                }

                return pager.From - pager.Limit
        case Backward:
                // ...
        }
        
        return -1
}
```